### DISCUSSION TITLE
Adding a credit card without making an order

### DISCUSSION THREAD

**Discussion by igoraugustynski** [swellstores/community#449] [Help] [answered]
Hello,
how do I add a credit card to a user account without an order? Should I use card.createToken and then account.createCard? If so, how?
The documentation is lacking in this regard.

**Comment by igoraugustynski** [comment]
Also, I have noticed that when adding a CC using the Swell interface it performs a transaction for 1 EURO. This does not happen when using account.createCard().

**Reply by ericingram** [reply, marked as answer]
We added some more details to the docs on adding a CC: https://developers.swell.is/frontend-api/credit-card-management. It only works with certain gateways at this time (Stripe and Braintree).

The Euro thing might be specific to your gateway, as we don't have logic like that in Swell itself.
